src/uct/api/uct.h

```diff
@@ -1205,7 +1205,7 @@ struct uct_listener_params {
 
     /**
      * Backlog of incoming connection requests.
-     * If not specified, SOMAXCONN, as defined in <sys/socket.h>, will be used.
+     * If not specified, each CM component will select its own value.
```

Comment:
can we move the doc change to another PR?

Reply:
well we are changing the behavior of setting the backlog. for rdmacm, for example, the previous doc won't be correct any more...